mantis-client/src/main/java/io/mantisrx/client/MantisClient.java

```diff
@@ -207,7 +206,7 @@ public String getError() {
         return new SinkClientImpl<T>(jobId, sinkConnectionFunc, getSinkLocator(),
                 numSinkWrkrsSubject
                         .filter((jobSinkNumWorkers) -> jobId.equals(jobSinkNumWorkers.getJobId()))
-                        .map((jobSinkNumWorkers) -> jobSinkNumWorkers.getNumSinkWorkers()),
+                        .map(MasterClientWrapper.JobSinkNumWorkers::getNumSinkWorkers),
```

Comment:
I usually try to avoid this style because as soon as the class name changes, it results in a change in this file too unnecessarily. If it's a popular class (in this case, MantisClientWrapper is and will be replaced with another type in my rewrite), then the number of such references is high, resulting in many spurious changes.

Reply:
sounds good